update.desktop

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env xdg-open
 [Desktop Entry]
 Name=Update Pikdum's Tools
-Exec=curl https://raw.githubusercontent.com/pikdum/steam-deck/master/install.sh | bash -s --
+Exec=curl https://raw.githubusercontent.com/SirStig/Steam-deck-tests/main/install.sh | bash -s --
```

Comment:
**issue:** This will need to be reverted to point back to this repo instead.

Reply:
Yeah, I know. I just figured we could do that here.